modules/structs/mirror.go

```diff
@@ -15,12 +17,12 @@ type CreatePushMirrorOption struct {
 // PushMirror represents information of a push mirror
 // swagger:model
 type PushMirror struct {
-	RepoName       string `json:"repo_name"`
-	RemoteName     string `json:"remote_name"`
-	RemoteAddress  string `json:"remote_address"`
-	CreatedUnix    string `json:"created"`
-	LastUpdateUnix string `json:"last_update"`
-	LastError      string `json:"last_error"`
-	Interval       string `json:"interval"`
-	SyncOnCommit   bool   `json:"sync_on_commit"`
+	RepoName       string    `json:"repo_name"`
+	RemoteName     string    `json:"remote_name"`
+	RemoteAddress  string    `json:"remote_address"`
+	CreatedUnix    time.Time `json:"created"`
+	LastUpdateUnix time.Time `json:"last_update"`
```

Comment:
```suggestion
  // swagger:strfmt date-time
	CreatedUnix    time.Time `json:"created"`
	// swagger:strfmt date-time
	LastUpdateUnix *time.Time `json:"last_update"`
```

Reply:
LastUpdateUnix should be a pointer to time.Time because it could be null

Reply:
Oh yeah, I missed that sorry

Reply:
So should I change it back :) ?